src/common/motion/position.ts

```diff
@@ -108,7 +108,7 @@ export class Position extends vscode.Position {
   constructor(line: number, character: number) {
     super(line, character);
 
-    this._nonWordCharRegex = this.makeWordRegex(Position.NonWordCharacters);
+    this._nonWordCharRegex = this.makeUnicodeWordRegex(Position.NonWordCharacters);
```

Comment:
We don't need to address this on this particular PR, but we should move this initialization elsewhere such that is it statically constructed rather than having to create it over and over again...